cli_tools/gce_vm_image_import/importer/importer.go

```diff
@@ -292,7 +292,17 @@ func Run(clientID string, imageName string, dataDisk bool, osID string, customTr
 		oauth, ce, gcsLogsDisabled, cloudLogsDisabled, stdoutLogsDisabled, kmsKey, kmsKeyring,
 		kmsLocation, kmsProject, noExternalIP, userLabels, storageLocation, uefiCompatible); err != nil {
 
+		postProcessErrorForNetwork(err, network, w)
+
 		return w, err
 	}
 	return w, nil
 }
+
+func postProcessErrorForNetwork(err error, network string, w *daisy.Workflow) {
+	if derr, ok := err.(daisy.DError); ok {
+		if derr.ErrType() == "networkResourceDoesNotExist" && network == "" {
+			w.LogWorkflowInfo("Please specify --network flag.")
+		}
```

Comment:
Can you add a couple more sentences? Here's a potential starting point:

"The default VPC has been removed from your GCP project, so you will need to specify one with the --network flag. The import workflow uses the network for [TODO] . For more information about creating networks, see [TODO] ."

Also, will the user need to provide a subnet?

Reply:
If the (non-default) network provided via --network flag is in auto mode, providing subnetwork is not mandatory. If it's a custom network, it is.

Reply:
sgtm

Reply:
"A VPC network is required for running worker instances" - may be too detailed for the user, I'd propose "A VPC network is required for running image import". How do you think?

Reply:
sounds good to me -- I like that it clarifies that the VPC is only used during import, rather than a permanent decision that is somehow tied to the imported resource.